src/views/poi_panel.dot

```diff
@@ -1,7 +1,7 @@
 <div class="poi_panel {{= this.active ? '' : 'poi_panel--hidden' }}">
   <div class="poi_panel__content">
   {{? this.poi }}
-    <div class="poi_panel__description_container">
+    <div class="poi_panel__description_container" {{= click(this.center, this) }}>
```

Comment:
There is currently no way for the user to notice that a click has an effect here.  
Maybe we could simply change the cursor on hover ?

Reply:
fixed